build.sbt

```diff
@@ -437,7 +437,6 @@ lazy val actorTestkitTyped = akkaModule("akka-actor-testkit-typed")
   .dependsOn(actorTyped, testkit % "compile->compile;test->test")
   .settings(AutomaticModuleName.settings("akka.actor.testkit.typed"))
   .settings(Dependencies.actorTestkitTyped)
-  .disablePlugins(MimaPlugin)
```

Comment:
Why are we removing here?

Reply:
Even tho we don't support bin compat for testkit it is a good indicator that we broke source compatibility so we don't do it accidently